docs/platform-openrpc.json

```diff
@@ -0,0 +1,1208 @@
+            "description": "Get a reception address for the account and optionally trigger address verification on the device.",
+            "paramStructure": "by-name",
+            "tags": [
+                {
+                    "$ref": "#/components/tags/Accounts"
+                },
+                {
+                    "$ref": "#/components/tags/PermissionReadAccount"
+                }
+            ],
+            "params": [
+                {
+                    "name": "account_id",
+                    "required": true,
+                    "schema": {
+                        "$ref": "#/components/schemas/AccountId"
+                    }
+                },
+                {
+                    "name": "verify",
+                    "summary": "Set to true, to verify the address on device",
+                    "required": false,
+                    "schema": {
+                        "type": "boolean"
+                    }
+                }
+            ],
+            "result": {
+                "name": "address",
+                "summary": "An address suitable to receive funds",
+                "description": "An address suitable to receive funds",
+                "schema": {
+                    "$ref": "#/components/schemas/Address"
+                }
+            }
+        },
+        {
+            "name": "ledgerlive.account.send",
+            "summary": "Start Ledger Live send flow to send a transaction",
+            "description": "Start Ledger Live send flow to send a transaction",
+            "params": [
+                {
+                    "name": "account_uid",
+                    "required": true,
+                    "summary": "The account on which to perform the transaction",
+                    "description": "The account on which to perform the transaction",
+                    "schema": {
+                        "$ref": "#/components/schemas/AccountId"
+                    }
+                },
+                {
+                    "name": "transaction",
+                    "required": false,
+                    "summary": "Default values for the transaction",
+                    "description": "Default values for the transaction",
+                    "schema": {
+                        "$ref": "#/components/schemas/Transaction"
+                    }
+                }
+            ],
+            "result": {
+                "$ref": "#/components/schemas/UnitSuccessResponse"
+            },
+            "tags": [
+                {
+                    "$ref": "#/components/tags/Transactions"
+                },
+                {
+                    "$ref": "#/components/tags/PermissionTransaction"
+                }
+            ]
+        },
+        {
+            "name": "ledgerlive.account.estimate_fees",
+            "summary": "Estimate fees for a given transaction. The method returns an updated version of the transaction passed in parameter",
```

Comment:
So for a swap / send, current flow is the following:
1. Live calls exchange nano app to validate protobuf inputs and trigger a transaction with bitcoin nano app
2. Then Live communicates with BTC app to generate signed transaction and broadcast

I can see how to do number 1 with the current API (estimate_fees + curencies.get + exchange), but hiw would we do number 2?